### DISCUSSION TITLE
FR: Add ability to format values in frontend

### DISCUSSION THREAD

**Discussion by jsiemek** [home-assistant/frontend#7553] [Other feature requests]
We sometimes have to deal with big numbers like lux values, number of packets, consumption of energy, ect.
So it will be nice to display them in format: 123.123.455,32
Declaring additional template sensor for all of then is more than a bit wired.
So it will be nice to have possibility to specify formatting options for frontend.
Something like {{ '{0:,.2f}'.format(sensor.name) }} in card definition.

**Comment by KTibow** [comment]
I'm not so sure about templates, but maybe this could be put in the same group as #7808 because both are about number displaying.
